src/amuse/community/hermite/tests/test_hermite.py

```diff
@@ -21,6 +21,9 @@
 except ImportError:
     HAS_MATPLOTLIB = False
 
+## Import generic nbody tests
```

Comment:
A better comment here would explain that this tests whether we've implemented the standard interfaces correctly, thus explaining why we're importing this. It's clear already that we're importing nbody tests from the framework because you named this well, so no need to further explain that.